winterfell/webapp/src/js/formCtrl.js

```diff
@@ -20,10 +24,18 @@ app.controller("formCtrl", ['$scope', 'formRenderingService', 'formTemplateServi
         };
 
         $scope.onSubmit = function () {
-            $scope.render();
+            console.log($scope.fields);
+            console.log($scope.model);
+
+            //$scope.render();
         };
 
+        $scope.model = {};
         $scope.fields = formTemplateService[$routeParams.formId];
 
-        $scope.model = {};
+        for (var i=0; i<$scope.fields.length; i++) {
+            if ($scope.fields[i].key.indexOf('date_signed') !== -1) {
+                $scope.model[$scope.fields[i].key] = currentDate();
+            }
+        }
     }]);
```

Comment:
a real small nit, but just fyi: let's try to end all files with a new line.
http://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline